fastify.js

```diff
@@ -285,6 +285,10 @@ function fastify (options) {
     onClose: null,
     close: null,
     printPlugins: null,
+    // check existence of the plugin
```

Comment:
This comment really isn't adding any value.

Reply:
@jsumners 
Thanks. I removed it.
https://github.com/fastify/fastify/pull/3963/commits/64ca9aaac0c5e48051e0fe9654febbea9495f3bf